lib/genesis/helpers.go

```diff
@@ -328,58 +435,49 @@ func generateStorageValue(i interface{}, idx int) ([]byte, error) {
 		if err != nil {
 			return nil, err
 		}
-	case [][]interface{}:
-		// TODO: for members field in phragmenElection struct figure out the correct format for encoding value (#1866)
```

Comment:
This issue is still open. if this is deliberate, let's close that issue.
If this has been removed by mistake lets bring this back.

I see `phragmenElection` in polkadot codebase. I don't find this keyword in our codebase, but we don't have spec files for kusama, polkadot and westend.

Reply:
I've closed this issue #1866